README.md

```diff
@@ -46,7 +46,7 @@ Install `minikube` and `helm` using Homebrew, then start `minikube` and update h
 brew install minikube helm
 minikube start --vm=true --driver=hyperkit
 minikube addons enable ingress
-helm dependency update helm/wasp-open-api
+helm dependency update helm/openapi-merger
```

Comment:
I believe this section on `Helm/Kubernetes` is now redundent @dblane-digicatapult and should be removed?

Reply:
> I believe this section on `Helm/Kubernetes` is now redundent @dblane-digicatapult and should be removed?

Left it for now

Reply:
Yeah can nuke this